src/Views/Signin/index.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import SigninForm from '../../Components/Signin/SigninForm';
-import SigninHeader from '../../Components/Signin/SigninHeader';
-import SigninHero from '../../Components/Signin/SigninHero';
+import React from './node_modules/react';
+import SigninForm from '../../components/signin/signin-form';
+import SigninHeader from '../../components/signin/signin-header';
+import SigninHero from '../../components/signin/signin-hero';
 import { Container, Hidden, Row, Col } from 'react-grid-system';
-import styled from 'styled-components';
+import styled from './node_modules/styled-components';
```

Comment:
Remove `./node_modules/` since a folder called `node_modules` does not exist in the Signin folder. Or use full path to root `node_modules`.